OsmAnd/res/values/strings.xml

```diff
@@ -75,12 +75,12 @@
     <string name="backup_restore_now">Restore now</string>
     <string name="backup_setup">Set up backup</string>
     <string name="backup_welcome_back_descr">Restore data on this device using existing backups, or back up data to OsmAnd Cloud.</string>
-    <string name="backup_welcome_back">Welcome back!</string>
```

Comment:
Shouting at the user (!) fails miserably in some languages, and is likely carried on in translation.